scripts/qa_crowbarsetup.sh

```diff
@@ -3864,31 +3864,32 @@ function oncontroller_testsetup
 
     # run tests for Magnum bay deployment
     if iscloudver 7plus && [[ $want_magnum = 1 ]]; then
-        if ! magnum baymodel-show susek8sbaymodel > /dev/null 2>&1; then
-            safely magnum baymodel-create --name susek8sbaymodel \
-                --image-id SLE12SP1-JeOS-k8s-magnum \
+        if ! magnum cluster-template-show k8s_template > /dev/null 2>&1; then
+            safely magnum cluster-template-create --name k8s_template \
+                --image-id sles-openstack-magnum-kubernetes \
                 --keypair-id default \
                 --external-network-id floating \
                 --flavor-id m1.smaller \
                 --master-flavor-id m2.smaller \
                 --docker-volume-size 5 \
                 --network-driver flannel \
                 --coe kubernetes \
+                --master-lb-enabled \
```

Comment:
This is because deprecated `bay-create` command was deploying k8s cluster with LB on front